code/obj/item/fabdisk.dm

```diff
@@ -0,0 +1,29 @@
+obj/item/fabdisk
+	name = "ManuDisk: Empty"
+	desc = "A cartridge for data storage that can be inserted and removed from manufacturers to temporarily add recipes to a manufacturer."
```

Comment:
You refer to "cartridge" here, and the sprites look like cartridges, so consider changing references to `disk` in this PR to `cartridge`. Per conversation in Discord, `disk` to me implies something else entirely - in that there would be some compatibility with systems that interact with diskettes - so if you're not looking to work within that system with this feature it should be very clearly separated from it.

Reply:
Cartridge didnt have a proper ring to it, decided on the name ManuDrive instead, we dont have any major things that are named drives (only one that comes to mine is storage drives for making termos computers), and also it still sounds nice on the tongue. (I really hope this name is sufficient because I just spent like 40 minutes renaming everything)